apps/dashboard/src/lib/components/Course/components/Lesson/Exercise/constants.js

```diff
@@ -6,15 +6,15 @@ export const STATUS = {
 
 export const SELECTABLE_STATUS = [
   {
-    label: 'Submitted',
-    value: STATUS.SUBMITTED
+    id: STATUS.SUBMITTED,
+    text: 'Submitted'
   },
   {
-    label: 'In progress',
-    value: STATUS.IN_PROGRESS
+    id: STATUS.IN_PROGRESS,
+    text: 'In progress'
   },
   {
-    label: 'Graded',
-    value: STATUS.GRADED
+    id: STATUS.GRADED,
+    text: 'Graded'
   }
```

Comment:
Use translation keys for `text` here instead.

Then in the place where it is rendered make sure to wrap it around `$t`. Make sure to see how it is done in other places